File: aggregator-pipeline/src/test/resources/net/shibboleth/metadata/pipeline/ScriptletStage-script.py
# Add a TestMarker to each item's item metadata
# "foo 0" for the first, "foo 1" for the second, etc.

from net.shibboleth.metadata import TestMarker;

count = 0
for item in items:
    marker = TestMarker('foo {}'.format(count))
    item.getItemMetadata().put(marker)
    count += 1

File: aggregator-pipeline/src/test/resources/net/shibboleth/metadata/pipeline/ScriptletStage-script.js
// Add a TestMarker to each item's item metadata
// "foo 0" for the first, "foo 1" for the second, etc.

var TestMarker = Java.type('net.shibboleth.metadata.TestMarker')
for (i=0; i<items.length; i++) {
    item = items[i]
    marker = new TestMarker('foo ' + i);
    item.getItemMetadata().put(marker);
}
